app/views/bars/show.html.haml

```diff
@@ -13,16 +13,17 @@
       %br
 
       %small
-        created by: tutaj dodac usera po polaczeniu modeli
+        created by:
+        = @bar.username
         =time_ago_in_words(@bar.created_at)
         ago   ||
         Last updated:
         =time_ago_in_words(@bar.updated_at)
         ago
-      %div
-        =link_to "Edit", edit_bar_path(@bar), class: "btn btn-success"
-        =link_to "Delete", bar_path(@bar), method: :delete,
-                    data: {confirm: "Are you sure you want to delete this bar?"},
-                    class: "btn btn-danger"
-      = render "image/new", bar: @bar
+      -if current_user == @bar.user
+        %div
+          =link_to "Edit", edit_bar_path(@bar), class: "btn btn-success"
+          =link_to "Delete", bar_path(@bar), method: :delete,
+                      data: {confirm: "Are you sure you want to delete this bar?"},
+                      class: "btn btn-danger"
```

Comment:
Please use https://github.com/airbnb/ruby style guide across files. Meaning empty spaces before `link_to` and `time_ago..` and after `{` / before `}`